University-advisor/SqlDriver.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Data.SQLite;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace University_advisor
+{
+    class SqlDriver
+    {
+        private static SQLiteConnection connect()
+        {
+            try
+            {
+                SQLiteConnection dbConnection = new SQLiteConnection("Data Source=Database.sqlite;Version=3;");
+                dbConnection.Open();
+                return dbConnection;
+            }
+            catch (SQLiteException e)
+            {
+                Console.WriteLine(e.StackTrace);
```

Comment:
As a temporary solution for showing exceptions it might work but maybe we could implement some kind of event logger in the future?